src/mpc-hc/mpc-hc.rc

```diff
@@ -2678,7 +2712,7 @@ BEGIN
     IDS_OSD_RS_VSYNC_OFFSET "VSync Offset: %d"
     IDS_OSD_SPEED           "Speed: %.2lfx"
     IDS_OSD_THUMBS_SAVED    "Thumbnails saved successfully"
-    IDS_MENU_VIDEO_STREAM   "&Video Stream"
+    IDS_MENU_VIDEO_STREAM   "&Video Streams"

```

Comment:
Revert this. We don't want to lose translations. Like I said we can change it but in different patch.